[package] libx265/3.4: Failed to build for Android

### Description

Failed to build "ffmpeg" for Android


### Package and Environment Details

* Package Name/Version: **libx265/3.4**
* Operating System: **Android**
* Compiler: **Clang 17**
* Conan version: **conan 2.2.3**
* Python version: **Python 3.12**
* Android NDK: **r26c**


### Conan profile

```text
[settings]
arch=armv8
os=Android
os.api_level=32

compiler=clang
compiler.version=17
compiler.libcxx=c++_static
compiler.cppstd=17

[options]
# Workaround: https://github.com/conan-io/conan-center-index/issues/20511
ffmpeg/*:with_libfdk_aac=False

# Workaround: https://github.com/conan-io/conan-center-index/issues/23718
ffmpeg/*:with_libx264=False

# https://docs.conan.io/2/reference/config_files/global_conf.html
[conf]
tools.cmake.cmaketoolchain:generator=Ninja

[tool_requires]
android-ndk/r26c
```

### Steps to reproduce

[Here](https://github.com/arttet/reproduce-bugs/tree/bug/conan-android-libx265) is a minimal example to reproduce the bug.

### Logs

<details><summary>Click to expand log</summary>

```
2024-04-23T15:27:30.6877548Z -------- Installing package libx265/3.4 (15 of 32) --------
2024-04-23T15:27:30.6878361Z libx265/3.4: Building from source
2024-04-23T15:27:30.6879170Z libx265/3.4: Package libx265/3.4:d6a7aa6ada37bcd989674ea21273b0a261e709d8
2024-04-23T15:27:30.6881883Z libx265/3.4: Copying sources to build folder
2024-04-23T15:27:30.7379193Z libx265/3.4: Building your package in /home/runner/.conan2/p/b/libx29962eaaf0f598/b
2024-04-23T15:27:30.7380342Z libx265/3.4: Calling generate()
2024-04-23T15:27:30.7381376Z libx265/3.4: Generators folder: /home/runner/.conan2/p/b/libx29962eaaf0f598/b/build/Debug/generators
2024-04-23T15:27:30.7700286Z libx265/3.4: CMakeToolchain generated: conan_toolchain.cmake
2024-04-23T15:27:30.7705109Z libx265/3.4: CMakeToolchain generated: CMakePresets.json
2024-04-23T15:27:30.7737419Z libx265/3.4: Generating aggregated env files
2024-04-23T15:27:30.7738753Z libx265/3.4: Generated aggregated env files: ['conanbuild.sh', 'conanrun.sh']
2024-04-23T15:27:30.7739652Z libx265/3.4: Calling build()
2024-04-23T15:27:30.7740658Z libx265/3.4: Apply patch (file): patches/3.2.1-0001-remove_register_classifier.patch
2024-04-23T15:27:30.7747659Z libx265/3.4: Apply patch (file): patches/3.2.1-0002-cmake-min-required.patch
2024-04-23T15:27:30.7755758Z libx265/3.4: Apply patch (file): patches/3.4-0001-numa.patch
2024-04-23T15:27:30.7778212Z libx265/3.4: Running CMake.configure()
2024-04-23T15:27:30.7780968Z libx265/3.4: RUN: cmake -G "Ninja" -DCMAKE_TOOLCHAIN_FILE="/home/runner/.conan2/p/b/libx29962eaaf0f598/b/build/Debug/generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="/home/runner/.conan2/p/b/libx29962eaaf0f598/p" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Debug" "/home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source"
2024-04-23T15:27:30.8061367Z CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
2024-04-23T15:27:30.8062668Z   Compatibility with CMake < 3.5 will be removed from a future version of
2024-04-23T15:27:30.8063465Z   CMake.
2024-04-23T15:27:30.8063670Z 
2024-04-23T15:27:30.8064100Z   Update the VERSION argument <min> value or use a ...<max> suffix to tell
2024-04-23T15:27:30.8065189Z   CMake that the project does not need compatibility with older versions.
2024-04-23T15:27:30.8065861Z 
2024-04-23T15:27:30.8066255Z 
2024-04-23T15:27:30.8066678Z -- cmake version 3.29.2
2024-04-23T15:27:30.8067314Z CMake Deprecation Warning at CMakeLists.txt:4 (cmake_policy):
2024-04-23T15:27:30.8068334Z   The OLD behavior for policy CMP0025 will be removed from a future version
2024-04-23T15:27:30.8069146Z   of CMake.
2024-04-23T15:27:30.8069370Z 
2024-04-23T15:27:30.8069848Z   The cmake-policies(7) manual explains that the OLD behaviors of all
2024-04-23T15:27:30.8070910Z   policies are deprecated and that a policy should be set to OLD only under
2024-04-23T15:27:30.8071683Z   specific short-term circumstances.  Projects should be ported to the NEW
2024-04-23T15:27:30.8072220Z   behavior and not rely on setting a policy to OLD.
2024-04-23T15:27:30.8072502Z 
2024-04-23T15:27:30.8072508Z 
2024-04-23T15:27:30.8072717Z CMake Deprecation Warning at CMakeLists.txt:10 (cmake_policy):
2024-04-23T15:27:30.8073301Z   The OLD behavior for policy CMP0054 will be removed from a future version
2024-04-23T15:27:30.8074009Z   of CMake.
2024-04-23T15:27:30.8074169Z 
2024-04-23T15:27:30.8074471Z   The cmake-policies(7) manual explains that the OLD behaviors of all
2024-04-23T15:27:30.8075075Z   policies are deprecated and that a policy should be set to OLD only under
2024-04-23T15:27:30.8075763Z   specific short-term circumstances.  Projects should be ported to the NEW
2024-04-23T15:27:30.8076288Z   behavior and not rely on setting a policy to OLD.
2024-04-23T15:27:30.8076809Z 
2024-04-23T15:27:30.8076815Z 
2024-04-23T15:27:30.8097681Z -- Using Conan toolchain: /home/runner/.conan2/p/b/libx29962eaaf0f598/b/build/Debug/generators/conan_toolchain.cmake
2024-04-23T15:27:30.8113990Z -- Conan toolchain: Setting CMAKE_POSITION_INDEPENDENT_CODE=ON (options.fPIC)
2024-04-23T15:27:30.8115130Z -- Conan toolchain: C++ Standard 17 with extensions OFF
2024-04-23T15:27:30.8116017Z -- Conan toolchain: Setting BUILD_SHARED_LIBS = OFF
2024-04-23T15:27:30.8411111Z -- The C compiler identification is Clang 17.0.2
2024-04-23T15:27:30.8797908Z -- The CXX compiler identification is Clang 17.0.2
2024-04-23T15:27:30.8999707Z -- Detecting C compiler ABI info
2024-04-23T15:27:30.9729362Z -- Detecting C compiler ABI info - done
2024-04-23T15:27:30.9911702Z -- Check for working C compiler: /home/runner/.conan2/p/andro0cd2cd114dfd6/p/bin/toolchains/llvm/prebuilt/linux-x86_64/bin/clang - skipped
2024-04-23T15:27:30.9915214Z -- Detecting C compile features
2024-04-23T15:27:30.9920532Z -- Detecting C compile features - done
2024-04-23T15:27:30.9991369Z -- Detecting CXX compiler ABI info
2024-04-23T15:27:31.0757631Z -- Detecting CXX compiler ABI info - done
2024-04-23T15:27:31.0937930Z -- Check for working CXX compiler: /home/runner/.conan2/p/andro0cd2cd114dfd6/p/bin/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++ - skipped
2024-04-23T15:27:31.0940733Z -- Detecting CXX compile features
2024-04-23T15:27:31.0946401Z -- Detecting CXX compile features - done
2024-04-23T15:27:31.0972063Z -- Cross compiling for ARM arch
2024-04-23T15:27:31.0972714Z -- Detected ARM64 target processor
2024-04-23T15:27:31.1108144Z -- Looking for include file inttypes.h
2024-04-23T15:27:31.1862332Z -- Looking for include file inttypes.h - found
2024-04-23T15:27:31.1863316Z -- cross compile arm
2024-04-23T15:27:31.1871564Z -- Performing Test CC_HAS_NO_STRICT_OVERFLOW
2024-04-23T15:27:31.2606497Z -- Performing Test CC_HAS_NO_STRICT_OVERFLOW - Success
2024-04-23T15:27:31.2613801Z -- Performing Test CC_HAS_NO_NARROWING
2024-04-23T15:27:31.3360666Z -- Performing Test CC_HAS_NO_NARROWING - Success
2024-04-23T15:27:31.3368806Z -- Performing Test CC_HAS_NO_ARRAY_BOUNDS
2024-04-23T15:27:31.4120428Z -- Performing Test CC_HAS_NO_ARRAY_BOUNDS - Success
2024-04-23T15:27:31.4128440Z -- Performing Test CC_HAS_FAST_MATH
2024-04-23T15:27:31.4891883Z -- Performing Test CC_HAS_FAST_MATH - Success
2024-04-23T15:27:31.4900097Z -- Performing Test CC_HAS_STACK_REALIGN
2024-04-23T15:27:31.5653414Z -- Performing Test CC_HAS_STACK_REALIGN - Success
2024-04-23T15:27:31.5661763Z -- Performing Test CC_HAS_FNO_EXCEPTIONS_FLAG
2024-04-23T15:27:31.6402115Z -- Performing Test CC_HAS_FNO_EXCEPTIONS_FLAG - Success
2024-04-23T15:27:31.6594691Z CMake Warning (dev) at /usr/local/share/cmake-3.29/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
2024-04-23T15:27:31.6596326Z   The package name passed to `find_package_handle_standard_args` (nasm) does
2024-04-23T15:27:31.6597660Z   not match the name of the calling package (Nasm).  This can lead to
2024-04-23T15:27:31.6599099Z   problems in calling code that expects `find_package` result variables
2024-04-23T15:27:31.6599988Z   (e.g., `_FOUND`) to follow a certain pattern.
2024-04-23T15:27:31.6600621Z Call Stack (most recent call first):
2024-04-23T15:27:31.6601346Z   cmake/FindNasm.cmake:23 (find_package_handle_standard_args)
2024-04-23T15:27:31.6602089Z   CMakeLists.txt:335 (find_package)
2024-04-23T15:27:31.6603031Z This warning is for project developers.  Use -Wno-dev to suppress it.
2024-04-23T15:27:31.6603675Z 
2024-04-23T15:27:31.6604014Z -- Could NOT find nasm (missing: NASM_EXECUTABLE) 
2024-04-23T15:27:31.8959954Z -- x265 version 0.0
2024-04-23T15:27:31.8975940Z -- Detected CXX compiler using -O3 optimization level
2024-04-23T15:27:31.8978662Z -- Looking for strtok_r
2024-04-23T15:27:31.9731802Z -- Looking for strtok_r - found
2024-04-23T15:27:32.0155762Z -- The ASM compiler identification is Clang with GNU-like command-line
2024-04-23T15:27:32.0196968Z -- Found assembler: /home/runner/.conan2/p/andro0cd2cd114dfd6/p/bin/toolchains/llvm/prebuilt/linux-x86_64/bin/clang
2024-04-23T15:27:32.0321571Z -- Looking for include file getopt.h
2024-04-23T15:27:32.1029888Z -- Looking for include file getopt.h - found
2024-04-23T15:27:32.1038181Z -- Configuring done (1.3s)
2024-04-23T15:27:32.1176784Z -- Generating done (0.0s)
2024-04-23T15:27:32.1248140Z -- Build files have been written to: /home/runner/.conan2/p/b/libx29962eaaf0f598/b/build/Debug
2024-04-23T15:27:32.1297169Z 
2024-04-23T15:27:32.1297631Z libx265/3.4: Running CMake.build()
2024-04-23T15:27:32.1300093Z libx265/3.4: RUN: cmake --build "/home/runner/.conan2/p/b/libx29962eaaf0f598/b/build/Debug" -- -j4
2024-04-23T15:27:32.6065313Z [1/68] Building CXX object encoder/CMakeFiles/encoder.dir/bitcost.cpp.o
2024-04-23T15:27:32.8896950Z [2/68] Building CXX object encoder/CMakeFiles/encoder.dir/motion.cpp.o
2024-04-23T15:27:33.0210684Z [3/68] Building CXX object encoder/CMakeFiles/encoder.dir/analysis.cpp.o
2024-04-23T15:27:33.0212778Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/analysis.cpp:33:
2024-04-23T15:27:33.0214819Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/analysis.h:30:
2024-04-23T15:27:33.0216772Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/predict.h:30:
2024-04-23T15:27:33.0218581Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/quant.h:29:
2024-04-23T15:27:33.0221580Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/scalinglist.h:36:12: warning: declaration shadows a variable in namespace 'x265' [-Wshadow]
2024-04-23T15:27:33.0223396Z     enum { NUM_SIZES = 4 };            // 4x4, 8x8, 16x16, 32x32
2024-04-23T15:27:33.0224056Z            ^
2024-04-23T15:27:33.0225126Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cudata.h:51:5: note: previous declaration is here
2024-04-23T15:27:33.0226351Z     NUM_SIZES
2024-04-23T15:27:33.0226710Z     ^
2024-04-23T15:27:33.0287581Z 1 warning generated.
2024-04-23T15:27:33.0813862Z [4/68] Building CXX object encoder/CMakeFiles/encoder.dir/search.cpp.o
2024-04-23T15:27:33.0816189Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/search.cpp:30:
2024-04-23T15:27:33.0818348Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/search.h:29:
2024-04-23T15:27:33.0820272Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/predict.h:30:
2024-04-23T15:27:33.0822388Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/quant.h:29:
2024-04-23T15:27:33.0825744Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/scalinglist.h:36:12: warning: declaration shadows a variable in namespace 'x265' [-Wshadow]
2024-04-23T15:27:33.0828034Z     enum { NUM_SIZES = 4 };            // 4x4, 8x8, 16x16, 32x32
2024-04-23T15:27:33.0829113Z            ^
2024-04-23T15:27:33.0830491Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cudata.h:51:5: note: previous declaration is here
2024-04-23T15:27:33.0832121Z     NUM_SIZES
2024-04-23T15:27:33.0832481Z     ^
2024-04-23T15:27:33.0834200Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/search.cpp:271:14: warning: variable 'tuDepthC' set but not used [-Wunused-but-set-variable]
2024-04-23T15:27:33.0835815Z     uint32_t tuDepthC = tuDepth;
2024-04-23T15:27:33.0836288Z              ^
2024-04-23T15:27:33.0838176Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/search.cpp:3905:14: warning: variable 'tuDepthC' set but not used [-Wunused-but-set-variable]
2024-04-23T15:27:33.0839864Z     uint32_t tuDepthC = tuDepth;
2024-04-23T15:27:33.0840410Z              ^
2024-04-23T15:27:33.0840835Z 3 warnings generated.
2024-04-23T15:27:33.3849095Z [5/68] Building CXX object encoder/CMakeFiles/encoder.dir/slicetype.cpp.o
2024-04-23T15:27:33.5535758Z [6/68] Building CXX object encoder/CMakeFiles/encoder.dir/level.cpp.o
2024-04-23T15:27:33.5844236Z [7/68] Building CXX object encoder/CMakeFiles/encoder.dir/frameencoder.cpp.o
2024-04-23T15:27:33.5846196Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/frameencoder.cpp:32:
2024-04-23T15:27:33.5848160Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/encoder.h:30:
2024-04-23T15:27:33.5850971Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/scalinglist.h:36:12: warning: declaration shadows a variable in namespace 'x265' [-Wshadow]
2024-04-23T15:27:33.5853124Z     enum { NUM_SIZES = 4 };            // 4x4, 8x8, 16x16, 32x32
2024-04-23T15:27:33.5854092Z            ^
2024-04-23T15:27:33.5855371Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cudata.h:51:5: note: previous declaration is here
2024-04-23T15:27:33.5856607Z     NUM_SIZES
2024-04-23T15:27:33.5856949Z     ^
2024-04-23T15:27:33.5858997Z 1 warning generated.
2024-04-23T15:27:33.5931689Z [8/68] Building CXX object encoder/CMakeFiles/encoder.dir/framefilter.cpp.o
2024-04-23T15:27:33.5933777Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/framefilter.cpp:28:
2024-04-23T15:27:33.5935954Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/encoder.h:30:
2024-04-23T15:27:33.5938950Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/scalinglist.h:36:12: warning: declaration shadows a variable in namespace 'x265' [-Wshadow]
2024-04-23T15:27:33.5941213Z     enum { NUM_SIZES = 4 };            // 4x4, 8x8, 16x16, 32x32
2024-04-23T15:27:33.5942141Z            ^
2024-04-23T15:27:33.5943355Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cudata.h:51:5: note: previous declaration is here
2024-04-23T15:27:33.5944590Z     NUM_SIZES
2024-04-23T15:27:33.5944934Z     ^
2024-04-23T15:27:33.5945274Z 1 warning generated.
2024-04-23T15:27:33.8143290Z [9/68] Building CXX object encoder/CMakeFiles/encoder.dir/nal.cpp.o
2024-04-23T15:27:34.0003070Z [10/68] Building CXX object encoder/CMakeFiles/encoder.dir/sei.cpp.o
2024-04-23T15:27:34.2266075Z [11/68] Building CXX object encoder/CMakeFiles/encoder.dir/sao.cpp.o
2024-04-23T15:27:34.2268552Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/sao.cpp:742:18: warning: variable 'fenc' set but not used [-Wunused-but-set-variable]
2024-04-23T15:27:34.2270272Z     const pixel* fenc;
2024-04-23T15:27:34.2270739Z                  ^
2024-04-23T15:27:34.2271167Z 1 warning generated.
2024-04-23T15:27:34.2333598Z [12/68] Building CXX object encoder/CMakeFiles/encoder.dir/entropy.cpp.o
2024-04-23T15:27:34.2336123Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/entropy.cpp:27:
2024-04-23T15:27:34.2339113Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/scalinglist.h:36:12: warning: declaration shadows a variable in namespace 'x265' [-Wshadow]
2024-04-23T15:27:34.2341232Z     enum { NUM_SIZES = 4 };            // 4x4, 8x8, 16x16, 32x32
2024-04-23T15:27:34.2342294Z            ^
2024-04-23T15:27:34.2344128Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cudata.h:51:5: note: previous declaration is here
2024-04-23T15:27:34.2347195Z     NUM_SIZES
2024-04-23T15:27:34.2347744Z     ^
2024-04-23T15:27:34.2349728Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/entropy.cpp:1361:21: warning: variable 'totalSignalledWeightFlags' set but not used [-Wunused-but-set-variable]
2024-04-23T15:27:34.2351887Z     uint32_t        totalSignalledWeightFlags = 0;
2024-04-23T15:27:34.2353079Z                     ^
2024-04-23T15:27:34.2353871Z 2 warnings generated.
2024-04-23T15:27:34.2840276Z [13/68] Building CXX object encoder/CMakeFiles/encoder.dir/dpb.cpp.o
2024-04-23T15:27:34.6840036Z [14/68] Building CXX object encoder/CMakeFiles/encoder.dir/reference.cpp.o
2024-04-23T15:27:34.7616662Z [15/68] Building CXX object encoder/CMakeFiles/encoder.dir/ratecontrol.cpp.o
2024-04-23T15:27:34.7618374Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/ratecontrol.cpp:36:
2024-04-23T15:27:34.7620205Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/encoder.h:30:
2024-04-23T15:27:34.7622909Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/scalinglist.h:36:12: warning: declaration shadows a variable in namespace 'x265' [-Wshadow]
2024-04-23T15:27:34.7624759Z     enum { NUM_SIZES = 4 };            // 4x4, 8x8, 16x16, 32x32
2024-04-23T15:27:34.7625489Z            ^
2024-04-23T15:27:34.7626649Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cudata.h:51:5: note: previous declaration is here
2024-04-23T15:27:34.7628012Z     NUM_SIZES
2024-04-23T15:27:34.7628393Z     ^
2024-04-23T15:27:34.7630290Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/ratecontrol.cpp:580:20: warning: variable 'totalQpAq' set but not used [-Wunused-but-set-variable]
2024-04-23T15:27:34.7632133Z             double totalQpAq = 0;
2024-04-23T15:27:34.7632680Z                    ^
2024-04-23T15:27:34.7634615Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/ratecontrol.cpp:996:32: warning: variable 'allCodedBits' set but not used [-Wunused-but-set-variable]
2024-04-23T15:27:34.7636755Z     uint64_t allConstBits = 0, allCodedBits = 0;
2024-04-23T15:27:34.7637435Z                                ^
2024-04-23T15:27:34.7639421Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/ratecontrol.cpp:1135:9: warning: variable 'iterations' set but not used [-Wunused-but-set-variable]
2024-04-23T15:27:34.7641274Z     int iterations = 0 , adjMin, adjMax;
2024-04-23T15:27:34.7641897Z         ^
2024-04-23T15:27:34.7642289Z 4 warnings generated.
2024-04-23T15:27:34.8631228Z [16/68] Building CXX object encoder/CMakeFiles/encoder.dir/api.cpp.o
2024-04-23T15:27:35.1973515Z [17/68] Building CXX object encoder/CMakeFiles/encoder.dir/weightPrediction.cpp.o
2024-04-23T15:27:35.2118322Z [18/68] Building CXX object common/CMakeFiles/common.dir/aarch64/asm-primitives.cpp.o
2024-04-23T15:27:35.3137571Z [19/68] Building CXX object common/CMakeFiles/common.dir/primitives.cpp.o
2024-04-23T15:27:35.4470494Z [20/68] Building CXX object encoder/CMakeFiles/encoder.dir/encoder.cpp.o
2024-04-23T15:27:35.4472490Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/encoder.cpp:36:
2024-04-23T15:27:35.4474451Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/encoder/encoder.h:30:
2024-04-23T15:27:35.4477606Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/scalinglist.h:36:12: warning: declaration shadows a variable in namespace 'x265' [-Wshadow]
2024-04-23T15:27:35.4480066Z     enum { NUM_SIZES = 4 };            // 4x4, 8x8, 16x16, 32x32
2024-04-23T15:27:35.4480982Z            ^
2024-04-23T15:27:35.4482263Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cudata.h:51:5: note: previous declaration is here
2024-04-23T15:27:35.4483743Z     NUM_SIZES
2024-04-23T15:27:35.4484289Z     ^
2024-04-23T15:27:35.4484822Z 1 warning generated.
2024-04-23T15:27:35.7653498Z [21/68] Building CXX object common/CMakeFiles/common.dir/lowpassdct.cpp.o
2024-04-23T15:27:35.7782462Z [22/68] Building CXX object common/CMakeFiles/common.dir/dct.cpp.o
2024-04-23T15:27:36.0960869Z [23/68] Building CXX object common/CMakeFiles/common.dir/pixel.cpp.o
2024-04-23T15:27:36.2274700Z [24/68] Building CXX object common/CMakeFiles/common.dir/loopfilter.cpp.o
2024-04-23T15:27:36.2287148Z [25/68] Building CXX object common/CMakeFiles/common.dir/intrapred.cpp.o
2024-04-23T15:27:36.5351785Z [26/68] Building CXX object common/CMakeFiles/common.dir/constants.cpp.o
2024-04-23T15:27:36.6368838Z [27/68] Building CXX object common/CMakeFiles/common.dir/cpu.cpp.o
2024-04-23T15:27:36.6371557Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cpu.cpp:335:26: warning: unused parameter 'benableavx512' [-Wunused-parameter]
2024-04-23T15:27:36.6373353Z uint32_t cpu_detect(bool benableavx512)
2024-04-23T15:27:36.6374079Z                          ^
2024-04-23T15:27:36.6375960Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cpu.cpp:48:13: warning: unused function 'sigill_handler' [-Wunused-function]
2024-04-23T15:27:36.6377664Z static void sigill_handler(int sig)
2024-04-23T15:27:36.6378337Z             ^
2024-04-23T15:27:36.6380054Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cpu.cpp:63:13: warning: unused variable 'enable512' [-Wunused-variable]
2024-04-23T15:27:36.6381688Z static bool enable512 = false;
2024-04-23T15:27:36.6382306Z             ^
2024-04-23T15:27:36.6382802Z 3 warnings generated.
2024-04-23T15:27:36.6631333Z [28/68] Building CXX object common/CMakeFiles/common.dir/version.cpp.o
2024-04-23T15:27:36.7530354Z [29/68] Building CXX object common/CMakeFiles/common.dir/ipfilter.cpp.o
2024-04-23T15:27:36.9785918Z [30/68] Building CXX object common/CMakeFiles/common.dir/threading.cpp.o
2024-04-23T15:27:37.1072965Z [31/68] Building CXX object common/CMakeFiles/common.dir/threadpool.cpp.o
2024-04-23T15:27:37.1114496Z [32/68] Building CXX object common/CMakeFiles/common.dir/wavefront.cpp.o
2024-04-23T15:27:37.2065539Z [33/68] Building CXX object common/CMakeFiles/common.dir/md5.cpp.o
2024-04-23T15:27:37.4301792Z [34/68] Building CXX object common/CMakeFiles/common.dir/bitstream.cpp.o
2024-04-23T15:27:37.5678500Z [35/68] Building CXX object common/CMakeFiles/common.dir/shortyuv.cpp.o
2024-04-23T15:27:37.5703825Z [36/68] Building CXX object common/CMakeFiles/common.dir/yuv.cpp.o
2024-04-23T15:27:37.6815095Z [37/68] Building CXX object common/CMakeFiles/common.dir/picyuv.cpp.o
2024-04-23T15:27:37.8814262Z [38/68] Building CXX object common/CMakeFiles/common.dir/common.cpp.o
2024-04-23T15:27:38.0575408Z [39/68] Building CXX object common/CMakeFiles/common.dir/frame.cpp.o
2024-04-23T15:27:38.1483106Z [40/68] Building CXX object common/CMakeFiles/common.dir/framedata.cpp.o
2024-04-23T15:27:38.2524580Z [41/68] Building CXX object common/CMakeFiles/common.dir/param.cpp.o
2024-04-23T15:27:38.2527013Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/param.cpp:785:16: warning: variable 'count' set but not used [-Wunused-but-set-variable]
2024-04-23T15:27:38.2528773Z     static int count;
2024-04-23T15:27:38.2529239Z                ^
2024-04-23T15:27:38.2529669Z 1 warning generated.
2024-04-23T15:27:38.4984072Z [42/68] Building CXX object common/CMakeFiles/common.dir/cudata.cpp.o
2024-04-23T15:27:38.4986447Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cudata.cpp:76:12: warning: use of bitwise '|' with boolean operands [-Wbitwise-instead-of-logical]
2024-04-23T15:27:38.4988203Z     return isEqualCol(addrA, addrB) | isEqualRow(addrA, addrB);
2024-04-23T15:27:38.4989418Z            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2024-04-23T15:27:38.4990045Z                                     ||
2024-04-23T15:27:38.4991552Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cudata.cpp:76:12: note: cast one or both operands to int to silence this warning
2024-04-23T15:27:38.4993146Z 1 warning generated.
2024-04-23T15:27:38.5338621Z [43/68] Building CXX object common/CMakeFiles/common.dir/slice.cpp.o
2024-04-23T15:27:38.6185301Z [44/68] Building CXX object common/CMakeFiles/common.dir/lowres.cpp.o
2024-04-23T15:27:38.7089513Z [45/68] Building CXX object common/CMakeFiles/common.dir/piclist.cpp.o
2024-04-23T15:27:38.9926966Z [46/68] Building CXX object common/CMakeFiles/common.dir/scalinglist.cpp.o
2024-04-23T15:27:39.0511302Z [47/68] Building CXX object common/CMakeFiles/common.dir/predict.cpp.o
2024-04-23T15:27:39.0513454Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/predict.cpp:29:
2024-04-23T15:27:39.0515573Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/predict.h:30:
2024-04-23T15:27:39.0517959Z In file included from /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/quant.h:29:
2024-04-23T15:27:39.0520971Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/scalinglist.h:36:12: warning: declaration shadows a variable in namespace 'x265' [-Wshadow]
2024-04-23T15:27:39.0523248Z     enum { NUM_SIZES = 4 };            // 4x4, 8x8, 16x16, 32x32
2024-04-23T15:27:39.0524324Z            ^
2024-04-23T15:27:39.0526188Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/cudata.h:51:5: note: previous declaration is here
2024-04-23T15:27:39.0527794Z     NUM_SIZES
2024-04-23T15:27:39.0528522Z     ^
2024-04-23T15:27:39.0530138Z 1 warning generated.
2024-04-23T15:27:39.1402778Z [48/68] Building CXX object common/CMakeFiles/common.dir/deblock.cpp.o
2024-04-23T15:27:39.3140119Z [49/68] Building CXX object common/CMakeFiles/common.dir/quant.cpp.o
2024-04-23T15:27:39.4094899Z [50/68] Building CXX object common/CMakeFiles/common.dir/scaler.cpp.o
2024-04-23T15:27:39.4097200Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/scaler.cpp:514:9: warning: variable 'dst_stride' set but not used [-Wunused-but-set-variable]
2024-04-23T15:27:39.4098445Z     int dst_stride = SCALER_ALIGN(dstW * sizeof(int16_t) + 66, 16);
2024-04-23T15:27:39.4098951Z         ^
2024-04-23T15:27:39.4099195Z 1 warning generated.
2024-04-23T15:27:39.4468681Z [51/68] Generating asm.S.o
2024-04-23T15:27:39.4638936Z [52/68] Generating mc-a.S.o
2024-04-23T15:27:39.4644105Z FAILED: mc-a.S.o /home/runner/.conan2/p/b/libx29962eaaf0f598/b/build/Debug/mc-a.S.o 
2024-04-23T15:27:39.4651145Z cd /home/runner/.conan2/p/b/libx29962eaaf0f598/b/build/Debug && /home/runner/.conan2/p/andro0cd2cd114dfd6/p/bin/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++ -fPIC -c /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S -o mc-a.S.o
2024-04-23T15:27:39.4663845Z <instantiation>:11:9: error: unknown directive
2024-04-23T15:27:39.4664832Z         .func x265_pixel_avg_pp_4x4_neon
2024-04-23T15:27:39.4666507Z         ^
2024-04-23T15:27:39.4667200Z <instantiation>:1:1: note: while in macro instantiation
2024-04-23T15:27:39.4668041Z function x265_pixel_avg_pp_4x4_neon
2024-04-23T15:27:39.4668802Z ^
2024-04-23T15:27:39.4672408Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4673903Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4674594Z ^
2024-04-23T15:27:39.4677610Z <instantiation>:1:6: error: unknown token in expression
2024-04-23T15:27:39.4678466Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4679056Z      ^
2024-04-23T15:27:39.4679809Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4682909Z .rept 4
2024-04-23T15:27:39.4683664Z ^
2024-04-23T15:27:39.4685355Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4687497Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4688172Z ^
2024-04-23T15:27:39.4688962Z <instantiation>:2:10: error: unknown token in expression
2024-04-23T15:27:39.4689846Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4690491Z          ^
2024-04-23T15:27:39.4691143Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4692256Z .rept 4
2024-04-23T15:27:39.4692630Z ^
2024-04-23T15:27:39.4693928Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4695218Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4695618Z ^
2024-04-23T15:27:39.4696233Z <instantiation>:3:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4696977Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4697439Z     ^~~~~~
2024-04-23T15:27:39.4697908Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4698551Z .rept 4
2024-04-23T15:27:39.4698867Z ^
2024-04-23T15:27:39.4700112Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4701400Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4701809Z ^
2024-04-23T15:27:39.4702236Z <instantiation>:4:10: error: unknown token in expression
2024-04-23T15:27:39.4702846Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4703250Z          ^
2024-04-23T15:27:39.4703742Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4704355Z .rept 4
2024-04-23T15:27:39.4704663Z ^
2024-04-23T15:27:39.4705896Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4707196Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4707606Z ^
2024-04-23T15:27:39.4708052Z <instantiation>:5:6: error: unknown token in expression
2024-04-23T15:27:39.4708691Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4709084Z      ^
2024-04-23T15:27:39.4709531Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4710136Z .rept 4
2024-04-23T15:27:39.4710436Z ^
2024-04-23T15:27:39.4711644Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4712887Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4713281Z ^
2024-04-23T15:27:39.4713742Z <instantiation>:6:10: error: unknown token in expression
2024-04-23T15:27:39.4714398Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4714822Z          ^
2024-04-23T15:27:39.4715282Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4715886Z .rept 4
2024-04-23T15:27:39.4716188Z ^
2024-04-23T15:27:39.4717611Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4718885Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4719289Z ^
2024-04-23T15:27:39.4719917Z <instantiation>:7:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4720701Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4721151Z     ^~~~~~
2024-04-23T15:27:39.4721614Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4722224Z .rept 4
2024-04-23T15:27:39.4722535Z ^
2024-04-23T15:27:39.4723749Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4725009Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4725398Z ^
2024-04-23T15:27:39.4725851Z <instantiation>:8:10: error: unknown token in expression
2024-04-23T15:27:39.4726491Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4726910Z          ^
2024-04-23T15:27:39.4727378Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4727988Z .rept 4
2024-04-23T15:27:39.4728294Z ^
2024-04-23T15:27:39.4729497Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4731098Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4731510Z ^
2024-04-23T15:27:39.4731961Z <instantiation>:9:6: error: unknown token in expression
2024-04-23T15:27:39.4732603Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4733006Z      ^
2024-04-23T15:27:39.4733467Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4734088Z .rept 4
2024-04-23T15:27:39.4734401Z ^
2024-04-23T15:27:39.4735932Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4737222Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4737627Z ^
2024-04-23T15:27:39.4738095Z <instantiation>:10:10: error: unknown token in expression
2024-04-23T15:27:39.4738759Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4739190Z          ^
2024-04-23T15:27:39.4739666Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4740287Z .rept 4
2024-04-23T15:27:39.4740596Z ^
2024-04-23T15:27:39.4741845Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4743141Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4743545Z ^
2024-04-23T15:27:39.4744182Z <instantiation>:11:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4744928Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4745377Z     ^~~~~~
2024-04-23T15:27:39.4745848Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4746481Z .rept 4
2024-04-23T15:27:39.4746792Z ^
2024-04-23T15:27:39.4748031Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4749310Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4749729Z ^
2024-04-23T15:27:39.4750198Z <instantiation>:12:10: error: unknown token in expression
2024-04-23T15:27:39.4750863Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4751293Z          ^
2024-04-23T15:27:39.4751761Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4752386Z .rept 4
2024-04-23T15:27:39.4752694Z ^
2024-04-23T15:27:39.4753924Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4755205Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4755613Z ^
2024-04-23T15:27:39.4757222Z <instantiation>:13:6: error: unknown token in expression
2024-04-23T15:27:39.4757907Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4758314Z      ^
2024-04-23T15:27:39.4758778Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4759415Z .rept 4
2024-04-23T15:27:39.4759718Z ^
2024-04-23T15:27:39.4760981Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4762270Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4762666Z ^
2024-04-23T15:27:39.4763137Z <instantiation>:14:10: error: unknown token in expression
2024-04-23T15:27:39.4763821Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4764255Z          ^
2024-04-23T15:27:39.4764731Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4765364Z .rept 4
2024-04-23T15:27:39.4765664Z ^
2024-04-23T15:27:39.4766914Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4768213Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4768610Z ^
2024-04-23T15:27:39.4769255Z <instantiation>:15:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4770011Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4770460Z     ^~~~~~
2024-04-23T15:27:39.4770936Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4771574Z .rept 4
2024-04-23T15:27:39.4771878Z ^
2024-04-23T15:27:39.4773121Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4774403Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4774796Z ^
2024-04-23T15:27:39.4775551Z <instantiation>:16:10: error: unknown token in expression
2024-04-23T15:27:39.4776226Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4776649Z          ^
2024-04-23T15:27:39.4777127Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4777753Z .rept 4
2024-04-23T15:27:39.4778054Z ^
2024-04-23T15:27:39.4779302Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4780887Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4781279Z ^
2024-04-23T15:27:39.4781671Z <instantiation>:2:9: error: unknown directive
2024-04-23T15:27:39.4782245Z         .endfunc
2024-04-23T15:27:39.4782594Z         ^
2024-04-23T15:27:39.4783065Z <instantiation>:9:1: note: while in macro instantiation
2024-04-23T15:27:39.4783688Z endfunc
2024-04-23T15:27:39.4783985Z ^
2024-04-23T15:27:39.4785227Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:44:1: note: while in macro instantiation
2024-04-23T15:27:39.4786525Z pixel_avg_pp_4xN_neon 4
2024-04-23T15:27:39.4786917Z ^
2024-04-23T15:27:39.4787310Z <instantiation>:11:9: error: unknown directive
2024-04-23T15:27:39.4787933Z         .func x265_pixel_avg_pp_4x8_neon
2024-04-23T15:27:39.4788443Z         ^
2024-04-23T15:27:39.4788910Z <instantiation>:1:1: note: while in macro instantiation
2024-04-23T15:27:39.4789592Z function x265_pixel_avg_pp_4x8_neon
2024-04-23T15:27:39.4790084Z ^
2024-04-23T15:27:39.4791320Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4792605Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4793001Z ^
2024-04-23T15:27:39.4793450Z <instantiation>:1:6: error: unknown token in expression
2024-04-23T15:27:39.4794110Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4794492Z      ^
2024-04-23T15:27:39.4794949Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4795577Z .rept 8
2024-04-23T15:27:39.4795881Z ^
2024-04-23T15:27:39.4797307Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4798627Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4799023Z ^
2024-04-23T15:27:39.4799487Z <instantiation>:2:10: error: unknown token in expression
2024-04-23T15:27:39.4800150Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4800571Z          ^
2024-04-23T15:27:39.4801046Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4801684Z .rept 8
2024-04-23T15:27:39.4801986Z ^
2024-04-23T15:27:39.4803228Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4804588Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4804980Z ^
2024-04-23T15:27:39.4805602Z <instantiation>:3:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4806350Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4806797Z     ^~~~~~
2024-04-23T15:27:39.4807266Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4807900Z .rept 8
2024-04-23T15:27:39.4808199Z ^
2024-04-23T15:27:39.4809434Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4810715Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4811112Z ^
2024-04-23T15:27:39.4811562Z <instantiation>:4:10: error: unknown token in expression
2024-04-23T15:27:39.4812236Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4812660Z          ^
2024-04-23T15:27:39.4813119Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4813747Z .rept 8
2024-04-23T15:27:39.4814053Z ^
2024-04-23T15:27:39.4815277Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4816570Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4816968Z ^
2024-04-23T15:27:39.4817409Z <instantiation>:5:6: error: unknown token in expression
2024-04-23T15:27:39.4818060Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4818705Z      ^
2024-04-23T15:27:39.4819157Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4819789Z .rept 8
2024-04-23T15:27:39.4820097Z ^
2024-04-23T15:27:39.4821339Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4822632Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4823255Z ^
2024-04-23T15:27:39.4823704Z <instantiation>:6:10: error: unknown token in expression
2024-04-23T15:27:39.4824367Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4824792Z          ^
2024-04-23T15:27:39.4825256Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4825888Z .rept 8
2024-04-23T15:27:39.4826192Z ^
2024-04-23T15:27:39.4827429Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4828713Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4829114Z ^
2024-04-23T15:27:39.4829746Z <instantiation>:7:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4830492Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4830947Z     ^~~~~~
2024-04-23T15:27:39.4831411Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4832060Z .rept 8
2024-04-23T15:27:39.4832369Z ^
2024-04-23T15:27:39.4833597Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4834888Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4835291Z ^
2024-04-23T15:27:39.4835748Z <instantiation>:8:10: error: unknown token in expression
2024-04-23T15:27:39.4836585Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4837021Z          ^
2024-04-23T15:27:39.4837493Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4838132Z .rept 8
2024-04-23T15:27:39.4838440Z ^
2024-04-23T15:27:39.4839695Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4841003Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4841401Z ^
2024-04-23T15:27:39.4841845Z <instantiation>:9:6: error: unknown token in expression
2024-04-23T15:27:39.4842497Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4842890Z      ^
2024-04-23T15:27:39.4843340Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4843968Z .rept 8
2024-04-23T15:27:39.4844276Z ^
2024-04-23T15:27:39.4845528Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4846821Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4847220Z ^
2024-04-23T15:27:39.4847680Z <instantiation>:10:10: error: unknown token in expression
2024-04-23T15:27:39.4848353Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4848783Z          ^
2024-04-23T15:27:39.4849245Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4849876Z .rept 8
2024-04-23T15:27:39.4850189Z ^
2024-04-23T15:27:39.4851430Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4852731Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4853136Z ^
2024-04-23T15:27:39.4853773Z <instantiation>:11:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4854535Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4854986Z     ^~~~~~
2024-04-23T15:27:39.4855460Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4856089Z .rept 8
2024-04-23T15:27:39.4856398Z ^
2024-04-23T15:27:39.4857625Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4858919Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4859319Z ^
2024-04-23T15:27:39.4859778Z <instantiation>:12:10: error: unknown token in expression
2024-04-23T15:27:39.4860447Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4860871Z          ^
2024-04-23T15:27:39.4861337Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4862230Z .rept 8
2024-04-23T15:27:39.4862543Z ^
2024-04-23T15:27:39.4863784Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4865082Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4865490Z ^
2024-04-23T15:27:39.4865957Z <instantiation>:13:6: error: unknown token in expression
2024-04-23T15:27:39.4866821Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4867220Z      ^
2024-04-23T15:27:39.4867685Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4868314Z .rept 8
2024-04-23T15:27:39.4868620Z ^
2024-04-23T15:27:39.4869860Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4871159Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4871561Z ^
2024-04-23T15:27:39.4872027Z <instantiation>:14:10: error: unknown token in expression
2024-04-23T15:27:39.4872686Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4873120Z          ^
2024-04-23T15:27:39.4873596Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4874228Z .rept 8
2024-04-23T15:27:39.4874538Z ^
2024-04-23T15:27:39.4875775Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4877616Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4878042Z ^
2024-04-23T15:27:39.4878693Z <instantiation>:15:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4879441Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4879899Z     ^~~~~~
2024-04-23T15:27:39.4880377Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4881000Z .rept 8
2024-04-23T15:27:39.4881307Z ^
2024-04-23T15:27:39.4882545Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4883830Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4884236Z ^
2024-04-23T15:27:39.4884709Z <instantiation>:16:10: error: unknown token in expression
2024-04-23T15:27:39.4885373Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4885799Z          ^
2024-04-23T15:27:39.4886274Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4886895Z .rept 8
2024-04-23T15:27:39.4887209Z ^
2024-04-23T15:27:39.4888459Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4889755Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4890156Z ^
2024-04-23T15:27:39.4890624Z <instantiation>:17:6: error: unknown token in expression
2024-04-23T15:27:39.4891278Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4891668Z      ^
2024-04-23T15:27:39.4892115Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4892739Z .rept 8
2024-04-23T15:27:39.4893049Z ^
2024-04-23T15:27:39.4894286Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4895571Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4895964Z ^
2024-04-23T15:27:39.4896428Z <instantiation>:18:10: error: unknown token in expression
2024-04-23T15:27:39.4897095Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4897524Z          ^
2024-04-23T15:27:39.4897995Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4898621Z .rept 8
2024-04-23T15:27:39.4898936Z ^
2024-04-23T15:27:39.4900169Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4901452Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4901855Z ^
2024-04-23T15:27:39.4902480Z <instantiation>:19:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4903228Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4903746Z     ^~~~~~
2024-04-23T15:27:39.4904224Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4904842Z .rept 8
2024-04-23T15:27:39.4905399Z ^
2024-04-23T15:27:39.4906649Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4907926Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4908329Z ^
2024-04-23T15:27:39.4908800Z <instantiation>:20:10: error: unknown token in expression
2024-04-23T15:27:39.4909457Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4909881Z          ^
2024-04-23T15:27:39.4910554Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4911183Z .rept 8
2024-04-23T15:27:39.4911488Z ^
2024-04-23T15:27:39.4912725Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4914005Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4914396Z ^
2024-04-23T15:27:39.4914857Z <instantiation>:21:6: error: unknown token in expression
2024-04-23T15:27:39.4915513Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4915892Z      ^
2024-04-23T15:27:39.4916357Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4917232Z .rept 8
2024-04-23T15:27:39.4917544Z ^
2024-04-23T15:27:39.4918803Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4920103Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4920498Z ^
2024-04-23T15:27:39.4920965Z <instantiation>:22:10: error: unknown token in expression
2024-04-23T15:27:39.4921646Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4922064Z          ^
2024-04-23T15:27:39.4922538Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4923173Z .rept 8
2024-04-23T15:27:39.4923475Z ^
2024-04-23T15:27:39.4924716Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4926004Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4926394Z ^
2024-04-23T15:27:39.4927021Z <instantiation>:23:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4927787Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4928236Z     ^~~~~~
2024-04-23T15:27:39.4928710Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4929339Z .rept 8
2024-04-23T15:27:39.4929644Z ^
2024-04-23T15:27:39.4930877Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4932165Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4932566Z ^
2024-04-23T15:27:39.4933029Z <instantiation>:24:10: error: unknown token in expression
2024-04-23T15:27:39.4933701Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4934121Z          ^
2024-04-23T15:27:39.4934593Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4935224Z .rept 8
2024-04-23T15:27:39.4935527Z ^
2024-04-23T15:27:39.4936753Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4938035Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4938429Z ^
2024-04-23T15:27:39.4938894Z <instantiation>:25:6: error: unknown token in expression
2024-04-23T15:27:39.4939562Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4939944Z      ^
2024-04-23T15:27:39.4940410Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4941038Z .rept 8
2024-04-23T15:27:39.4941340Z ^
2024-04-23T15:27:39.4942570Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4943869Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4944261Z ^
2024-04-23T15:27:39.4944723Z <instantiation>:26:10: error: unknown token in expression
2024-04-23T15:27:39.4945389Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4945806Z          ^
2024-04-23T15:27:39.4946277Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4946910Z .rept 8
2024-04-23T15:27:39.4947209Z ^
2024-04-23T15:27:39.4948435Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4950006Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4950399Z ^
2024-04-23T15:27:39.4951038Z <instantiation>:27:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4951793Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4952238Z     ^~~~~~
2024-04-23T15:27:39.4952717Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4953563Z .rept 8
2024-04-23T15:27:39.4953870Z ^
2024-04-23T15:27:39.4955116Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4956578Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4956988Z ^
2024-04-23T15:27:39.4957463Z <instantiation>:28:10: error: unknown token in expression
2024-04-23T15:27:39.4958132Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4958551Z          ^
2024-04-23T15:27:39.4959026Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4959659Z .rept 8
2024-04-23T15:27:39.4959976Z ^
2024-04-23T15:27:39.4961225Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4962512Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4962905Z ^
2024-04-23T15:27:39.4963369Z <instantiation>:29:6: error: unknown token in expression
2024-04-23T15:27:39.4964031Z ld1 {v0.s}[0], [x2], x3
2024-04-23T15:27:39.4964414Z      ^
2024-04-23T15:27:39.4964878Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4965511Z .rept 8
2024-04-23T15:27:39.4965811Z ^
2024-04-23T15:27:39.4967053Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4968345Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4968743Z ^
2024-04-23T15:27:39.4969204Z <instantiation>:30:10: error: unknown token in expression
2024-04-23T15:27:39.4969868Z     ld1 {v1.s}[0], [x4], x5
2024-04-23T15:27:39.4970294Z          ^
2024-04-23T15:27:39.4970764Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4971393Z .rept 8
2024-04-23T15:27:39.4971701Z ^
2024-04-23T15:27:39.4972934Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4974228Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4974623Z ^
2024-04-23T15:27:39.4975243Z <instantiation>:31:5: error: invalid instruction mnemonic 'urhadd'
2024-04-23T15:27:39.4976023Z     urhadd v2.8b, v0.8b, v1.8b
2024-04-23T15:27:39.4976472Z     ^~~~~~
2024-04-23T15:27:39.4976936Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4977567Z .rept 8
2024-04-23T15:27:39.4977874Z ^
2024-04-23T15:27:39.4979095Z /home/runner/.conan2/p/b/libx29962eaaf0f598/b/src/source/common/aarch64/mc-a.S:45:1: note: while in macro instantiation
2024-04-23T15:27:39.4980389Z pixel_avg_pp_4xN_neon 8
2024-04-23T15:27:39.4980790Z ^
2024-04-23T15:27:39.4981251Z <instantiation>:32:10: error: unknown token in expression
2024-04-23T15:27:39.4981951Z     st1 {v2.s}[0], [x0], x1
2024-04-23T15:27:39.4982367Z          ^
2024-04-23T15:27:39.4982831Z <instantiation>:2:1: note: while in macro instantiation
2024-04-23T15:27:39.4983463Z .rept 8
2024-04-23T15:27:39.4983771Z ^
```

</details>

--- a/recipes/libx265/all/conanfile.py
+++ b/recipes/libx265/all/conanfile.py
@@ -83,7 +83,7 @@ class Libx265Conan(ConanFile):
             # "x265::setupAssemblyPrimitives(x265::EncoderPrimitives&, int)", referenced from:
             # x265::x265_setup_primitives(x265_param*) in libx265.a[20](primitives.cpp.o)
             # ld: symbol(s) not found for architecture arm64
-            raise ConanInvalidConfiguration(f"{self.ref} does not support Mac M1.")
+            raise ConanInvalidConfiguration(f"{self.ref} fails to build for Mac M1. Contributions are welcome.")
 
     def build_requirements(self):
         if self.options.assembly:

--- a/recipes/libx265/all/conanfile.py
+++ b/recipes/libx265/all/conanfile.py
@@ -78,7 +78,7 @@ class Libx265Conan(ConanFile):
         if self.options.shared and is_msvc(self) and is_msvc_static_runtime(self):
             raise ConanInvalidConfiguration("shared not supported with static runtime")
 
-        if self.settings.compiler == "apple-clang" and "arm" in self.settings.arch:
+        if self.settings.compiler == "apple-clang" and "arm" in self.settings.arch and self.options.assembly:
             # Undefined symbols for architecture arm64:
             # "x265::setupAssemblyPrimitives(x265::EncoderPrimitives&, int)", referenced from:
             # x265::x265_setup_primitives(x265_param*) in libx265.a[20](primitives.cpp.o)

--- a/recipes/libx265/all/conanfile.py
+++ b/recipes/libx265/all/conanfile.py
@@ -78,6 +78,13 @@ class Libx265Conan(ConanFile):
         if self.options.shared and is_msvc(self) and is_msvc_static_runtime(self):
             raise ConanInvalidConfiguration("shared not supported with static runtime")
 
+        if self.settings.compiler == "apple-clang" and "arm" in self.settings.arch:
+            # Undefined symbols for architecture arm64:
+            # "x265::setupAssemblyPrimitives(x265::EncoderPrimitives&, int)", referenced from:
+            # x265::x265_setup_primitives(x265_param*) in libx265.a[20](primitives.cpp.o)
+            # ld: symbol(s) not found for architecture arm64
+            raise ConanInvalidConfiguration(f"{self.ref} does not support Mac M1.")
+
     def build_requirements(self):
         if self.options.assembly:
             if self.settings.arch in ["x86", "x86_64"]:

--- a/recipes/libx265/all/conanfile.py
+++ b/recipes/libx265/all/conanfile.py
@@ -49,7 +49,8 @@ class Libx265Conan(ConanFile):
             del self.options.with_numa
         # FIXME: Disable assembly by default if host is arm and compiler apple-clang for the moment.
         # Indeed, apple-clang is not able to understand some asm instructions of libx265
-        if self.settings.compiler == "apple-clang" and "arm" in self.settings.arch:
+        # FIXME: Disable assembly by default if host is Android for the moment. It fails to build
+        if (self.settings.compiler == "apple-clang" and "arm" in self.settings.arch) or self.settings.os == "Android":
             self.options.assembly = False
 
     def configure(self):
@@ -62,6 +63,16 @@ class Libx265Conan(ConanFile):
     def requirements(self):
         if self.options.get_safe("with_numa", False):
             self.requires("libnuma/2.0.14")
+
+    def validate_build(self):
+        if cross_building(self) and self.settings.os == "Android" and self.options.assembly:
+            # FIXME: x265 uses custom command to invoke clang to compile assembly files.
+            #   clang++ -fPIC -c src/source/common/aarch64/mc-a.S -o mc-a.S.o
+            #   FAILED: mc-a.S.o libx2f309356bd8526/b/build/Release/mc-a.S.o
+            #   clang++ -fPIC -c src/source/common/aarch64/mc-a.S -o mc-a.S.o
+            #   <instantiation>:11:9: error: unknown directive
+            #           .func x265_pixel_avg_pp_4x4_neon
+            raise ConanInvalidConfiguration(f"{self.ref} fails to build with '&:assembly=True' for Android. Contributions are welcome.")
 
     def validate(self):
         if self.options.shared and is_msvc(self) and is_msvc_static_runtime(self):
@@ -154,7 +165,10 @@ class Libx265Conan(ConanFile):
             if not self.options.shared:
                 self.cpp_info.sharedlinkflags = ["-Wl,-Bsymbolic,-znoexecstack"]
         elif self.settings.os == "Android":
-            self.cpp_info.libs.extend(["dl", "m"])
-        libcxx = stdcpp_library(self)
-        if libcxx:
-            self.cpp_info.system_libs.append(libcxx)
+            self.cpp_info.system_libs.extend(["dl", "m"])
+        if not self.options.shared:
+            libcxx = stdcpp_library(self)
+            if libcxx:
+                if self.settings.os == "Android" and self.settings.compiler.libcxx == "c++_static":
+                    self.cpp_info.system_libs.append("c++abi")
+                self.cpp_info.system_libs.append(libcxx)

